.github/settings.yml

```diff
@@ -1,40 +1,74 @@
-repository:
-  has_wiki: false
-
-labels:
-  - name: "duplicate"
-    color: cfd3d7
-  - name: "good first issue"
-    color: 7057ff
-  - name: "invalid"
-    color: cfd3d7
-  - name: "more data needed"
-    color: bfdadc
-  - name: "P0"
-    color: b60205
-    description: "blocker: fix immediately!"
-  - name: "P1"
-    color: d93f0b
-    description: "critical: next release"
-  - name: "P2"
-    color: e99695
-    description: "major: an upcoming release"
-  - name: "P3"
-    color: fbca04
-    description: "minor: not priorized"
-  - name: "P4"
-    color: fef2c0
-    description: "unimportant: consider wontfix or other priority"
-  - name: "question"
-    color: d876e3
-  - name: "type: bug"
-    color: 0052cc
-  - name: "type: documentation"
-    color: 0052cc
-  - name: "type: feature request"
-    color: 0052cc
-  - name: "wontfix"
-    color: ffffff
-  - name: "merge-queue"
-    color: 0e8a16
-    description: "merge on green CI"
+# DO NOT EDIT! Generated by eval-github-yaml from .github/settings.ncl
+{
+  "labels": [
+    {
+      "color": "cfd3d7",
+      "name": "duplicate"
+    },
+    {
+      "color": "7057ff",
+      "name": "good first issue"
+    },
+    {
+      "color": "cfd3d7",
+      "name": "invalid"
+    },
+    {
+      "color": "bfdadc",
+      "name": "more data needed"
+    },
+    {
+      "color": "b60205",
+      "description": "blocker: fix immediately!",
+      "name": "P0"
+    },
+    {
+      "color": "d93f0b",
+      "description": "critical: next release",
+      "name": "P1"
+    },
+    {
+      "color": "e99695",
+      "description": "major: an upcoming release",
+      "name": "P2"
+    },
+    {
+      "color": "fbca04",
+      "description": "minor: not priorized",
+      "name": "P3"
+    },
+    {
+      "color": "fef2c0",
+      "description": "unimportant: consider wontfix or other priority",
+      "name": "P4"
+    },
+    {
+      "color": "d876e3",
+      "name": "question"
+    },
+    {
+      "color": "0052cc",
+      "name": "type: bug"
+    },
+    {
+      "color": "0052cc",
+      "name": "type: documentation"
+    },
+    {
+      "color": "0052cc",
+      "name": "type: feature request"
+    },
+    {
+      "color": "ffffff",
+      "name": "wontfix"
+    },
+    {
+      "color": "0e8a16",
+      "description": "merge on green CI",
+      "name": "merge-queue"
+    }
+  ],
+  "repository": {
+    "has_wiki": false
+  }
+}
```

Comment:
Maybe make sure that there's a newline at the end of generated files as Git likes it better.

Reply:
Very good point; addressed by patching Nickel proper 👍